ui/static/ui/css/mit-lore.css

```diff
@@ -498,6 +498,32 @@ ul.with-utility-features {
     word-wrap: break-word;
 }
 
+div#term-list-container .panel-heading {
```

Comment:
Changed tabs to four spaces -- let me know if that fixed what you were seeing. Thanks.